arch/sparc/core/entry.S

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright (c) 2018 ispace, inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <toolchain.h>
+#include <autoconf.h>
+#include <arch/sparc/arch.h>
+
+/* exports */
+GTEXT(__start)
+
+/* imports */
+GTEXT(_PrepC)
```

Comment:
you need to namespace all your private variables and functions to this arch port with z_, preferably z_sparc_ for clarity

Reply:
done.